File: springboot-version/backend/src/main/java/com/petconnect/dto/NotificationDto.java
package com.petconnect.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class NotificationDto {
    private Long id;
    private String type;
    private String message;

    @com.fasterxml.jackson.annotation.JsonProperty("isRead")
    private boolean isRead;

    private Long relatedPostId;
    private Long senderId;
    private LocalDateTime createdAt;
}

File: aspnet-version/backend/PetConnectAlmost/DTOs/NotificationDto.cs
namespace PetConnectAlmost.DTOs
{
    public class NotificationDto
    {
        public long NotificationId { get; set; }
        public long UserId { get; set; }
        public string Type { get; set; } = string.Empty;
        public string Message { get; set; } = string.Empty;
        public bool IsRead { get; set; }
        public long? RelatedPostId { get; set; }
        public long? SenderId { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
